en/ros2/user_guide.md

```diff
@@ -451,9 +458,38 @@ No further action is required by the user.
 In this scenario, ROS2 also uses the Gazebo `/clock` topic as time source.
 This approach makes sense if the Gazebo simulation is running with real time factor different from one, or if ROS2 needs to directly interact with Gazebo.
 On the ROS2 side, direct interaction with Gazebo is achieved by the [ros_gz_bridge](https://github.com/gazebosim/ros_gz) package of the [ros_gz](https://github.com/gazebosim/ros_gz) repository.
-Read through the [repo](https://github.com/gazebosim/ros_gz#readme) and [package](https://github.com/gazebosim/ros_gz/tree/ros2/ros_gz_bridge#readme) READMEs to find out the right version that has to be installed depending on your ROS2 and Gazebo versions.
 
-Once the package is installed and sourced, the node `parameter_bridge` provides the bridging capabilities and can be used to create an unidirectional `/clock` bridge:
+Use the following commands to install the correct ROS 2/gz interface packages (not just the bridge) for the ROS2 and Gazebo version(s) supported by PX4.
+
+:::: tabs
+
+::: tab humble
+To install the bridge for use with ROS 2 "Humble" and Gazebo Garden (on Ubuntu 22.04):
+
+```sh
+sudo apt install ros-humble-ros-gzgarden
+```
+
+:::
+
+::: tab foxy
+First you will need to [install Gazebo Garden](../sim_gazebo_gz/index.md#installation-ubuntu-linux), as by default Foxy comes with Gazebo Classic 11.
+
+Then to install the interface packages for use with ROS 2 "Foxy" and Gazebo Garden (Ubuntu 20.04):
+
+```sh
+sudo apt install ros-foxy-ros-gzgarden
+```
+
+:::
+
+::::
+
+::: info
+The [repo](https://github.com/gazebosim/ros_gz#readme) and [package](https://github.com/gazebosim/ros_gz/tree/ros2/ros_gz_bridge#readme) READMEs show the package versions that need to be installed depending on your ROS2 and Gazebo versions.
+:::
+
+Once the packages are installed and sourced, the node `parameter_bridge` provides the bridging capabilities and can be used to create an unidirectional `/clock` bridge:
```

Comment:
@federicociresola @beniaminopozzan Looking at this, I think that there was an unnecessary risk in the instructions in pointing to the readmes to work out the correct versions of the bridge. We know the correct versions because we know the versions PX4 supports, so why not spell it out. 

So what I have done is added this section ^^^ which outlines the humble and foxy variants of binaries to install (@beniaminopozzan - specifying the interface package, not just the bridge, as you suggested). I added the old note as Info at the end so people can see that there are different versions that might work.

Reply:
For it is a good proposal, I would like to suggest to add a line with wrote a think like "if you have installed the gazebo from ubuntu.sh we suggest you to remove it and keep only the version installed from the ROS2 package so that there are no conflicts between versions"

What do @beniaminopozzan think about it?

Reply:
> What do @beniaminopozzan think about it?

Good question. Note, I am away until Wednesday, but I will respond then to whatever @beniaminopozzan says.

I think the suggestion to remove any conflicting version would be good, if it is possible to do.

Reply:
@beniaminopozzan You must have missed this request or been too busy. 

I am pretty sure that the rest of this is good. I'm going to merge, but if you could do a post-review that would be good.

@federicociresola Thanks for this. I have chosen not to add the note:

>  "if you have installed the gazebo from ubuntu.sh we suggest you to remove it and keep only the version installed from the ROS2 package so that there are no conflicts between versions"

Because there shouldn't be any conflict if you follow the instructions. But @beniaminopozzan we can add it he thinks it is needed.

Reply:
Perfect, I am happy to contribute to this amazing project when I am able

Reply:
Thanks @hamishwillee , apologises for missing this (again).
It looks very good to me, nice work!

Reply:
NO worries! Thanks for the post submit check!